web/beacon-app/src/features/projects/components/APIKeysTable.tsx

```diff
@@ -71,8 +79,35 @@ export const APIKeysTable = ({ projectID }: APIKeysTableProps) => {
         trClassName="text-sm"
         className="w-full"
         columns={[
-          { Header: 'Name', accessor: 'name' },
-          { Header: 'Client ID', accessor: 'client_id' },
+          { Header: 'Key Name', accessor: 'name' },
+          { Header: 'Permissions', accessor: 'permissions' },
+          {
+            Header: 'Status',
+            accessor: (key: { status: APIKeyStatus }) => {
+              return (
+                <div className="flex items-center">
+                  {key.status === APIKEY_STATUS.ACTIVE && <ConfirmedIndicator />}
+                  {key.status === APIKEY_STATUS.INACTIVE && <PendingIndicator />}
+                  {key.status === APIKEY_STATUS.REVOKED && <RevokedIndicator />}
+                  {key.status === APIKEY_STATUS.UNUSED && <UnusedIndicator />}
```

Comment:
I think here we can create an IconMap to avoid a long conditional statement
```ts
const statusIconMap = {
   [APIKEY_STATUS.ACTIVE]: <ConfirmedIndicator />,
   [APIKEY_STATUS.INACTIVE]: <PendingIndicator />,
   ...
}
```
and use it like that

```ts
statusIconMap[key.status]
```

```suggestion
                  {statusIconMap[key.status]}
```

Reply:
Thank you!